nglview/utils/widget_utils.py

```diff
@@ -90,3 +92,66 @@ def func(this, **kwargs):
             func = make_func(rep)
             fn = '_'.join((root_fn, rep[0]))
             setattr(self, fn, MethodType(func, other))
+
+
+def compare_two_images(image_path_1,
+                       image_path_2,
+                       standard_width=200,
+                       standard_height=200,
+                       convert_to_gray=False,
+                       threshold=0.97):
```

Comment:
I don't think we need those extra argument. e.g: why do we care about "200" here? 
may be keeping `threshold`